core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -69,13 +76,13 @@
 public class ArquillianCucumber extends Arquillian {
     private static final Logger LOGGER = Logger.getLogger(ArquillianCucumber.class.getName());
 
-    private static final String RUN_CUCUMBER_MTD = "____Cucumber_Runner_Not_A_Test";
```

Comment:
originally I voluntary wanted something not respecting java convention because of Intellij Idea cucumber integration which makes cucumber, normal test methods and this particular one really separated visually.

Maybe we should just get rid of it as a children and just call it when expected in the runner.

I dont have time to dig into it now but would be the best solution